tests/io/FileFactoryTest.py

```diff
@@ -0,0 +1,47 @@
+import os
+import unittest
+
+from coalib.io.FileFactory import FileFactory
+
+
+class FileFactoryTest(unittest.TestCase):
+
+    def setUp(self):
+        file_factory_test_dir = os.path.join(os.path.split(__file__)[0],
+                                             'FileFactoryTestFiles')
+
+        self.test_file = os.path.join(file_factory_test_dir, 'test.txt')
+        self.file_factory_object = FileFactory(self.test_file)
+
+    def test_equal(self):
+        self.assertEqual(self.file_factory_object,
+                         FileFactory(self.test_file))
+
+    def test_iter(self):
+        self.file_factory_object.lines
+        for f in self.file_factory_object:
+            self.assertEqual(type(f), str)
+
+    def test_line(self):
+        self.assertEqual(self.file_factory_object.get_line(0),
+                         'This is a test file.')
+
+    def test_lines(self):
+        file_tuple = self.file_factory_object.lines
+        self.assertEqual(file_tuple, ('This is a test file.',))
```

Comment:
I would inline `file_tuple`, then the test layout looks the same in all places ;)